### PR TITLE
Add tests for sklearnex package

### PR DESCRIPTION
**Pull request by Pahandrovich**

### CODE REVIEW

**Review by OnlyDeniko — APPROVED** [review]

**Comment by PetrovKP** [comment]
can we add to public CI?

File: sklearnex/tests/test_monkeypatch.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+import sklearnex
+from daal4py.sklearn import sklearn_patch_map
```
Comment (PetrovKP) [inline sklearnex/tests/test_monkeypatch.py:18]: daal4py?

Comment (Pahandrovich) [reply sklearnex/tests/test_monkeypatch.py:18]: this function is not available in sklearnex

Comment (PetrovKP) [reply sklearnex/tests/test_monkeypatch.py:18]: get_patch_names ? No?

File: sklearnex/tests/test_patching.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        if not mas[i].startswith('sklearn'):
+            ind = name + ' ' + dtype + ' ' + mas[i]
+            result[ind] = get_branch(temp)
+            temp.clear()
+        else:
+            temp.append(mas[i])
+
+
+def get_result_log():
+    os.environ['IDP_SKLEARN_VERBOSE'] = 'INFO'
```
Comment (PetrovKP) [inline sklearnex/tests/test_patching.py:53]: Fix it. We testing sklearnex package.

File: sklearnex/tests/_models_info.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    {
+        'model': KNeighborsRegressor(algorithm='brute'),
+        'methods': ['kneighbors', 'predict', 'score'],
+        'dataset': 'regression',
+    },
+    {
+        'model': NearestNeighbors(algorithm='brute'),
+        'methods': ['kneighbors'],
+        'dataset': 'blobs',
+    },
```
Comment (SmirnovEgorRu) [inline sklearnex/tests/_models_info.py:52]: kd-tree?

File: sklearnex/tests/utils/_launch_algorithms.py [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+        else:
+            raise ValueError(i + ' is wrong method')
+        logging.info(i)
+
+
+def run_algotithms():
+    for info in MODELS_INFO:
+        for t in TYPES:
+            model_name = get_class_name(info['model'])
+            if model_name in ['Ridge', 'LinearRegression'] and t.__name__ == 'uint32':
```
Comment (SmirnovEgorRu) [inline sklearnex/tests/utils/_launch_algorithms.py:91]: uint32 doesn't work for them, right?

File: sklearnex/tests/test_monkeypatch.py (hunk 1 above)
Comment (Pahandrovich) [reply sklearnex/tests/test_monkeypatch.py:18]: get_patch_names gets only **values**, but we need **keys** in this file

**Comment by PetrovKP** [comment]
@Mergifyio backport rls/2021.2-rls

**Comment by PetrovKP** [comment]
@Mergifyio backport rls/2021.2.1-rls

**Review by PetrovKP — APPROVED** [review]

**Pull request merged**